adafruit_displayio_layout/widgets/cartesian.py

```diff
@@ -609,6 +616,9 @@ def add_plot_line(self, x: int, y: int) -> None:
 
         rtype: None
         """
+        # mypy, plot_line_point can't be None at this point
+        assert self.plot_line_point is not None
```

Comment:
I think it's worth wider consideration and conversation about including the assert statements. If I'm understanding correctly, this code resolves an issue that mypy reports.

We haven't enabled automated mypy yet, and when we do we'll decide on a configuration used with regards to which warnings will constitute pass or fail of the actions run. 

It's worth considering whether we want to include asserts like this in order to quiet certain errors and warnings. In my understanding I think this line will add to the size of the mpy file and RAM used when the library is imported, both of which are fairly constrained on some devices. In contrast the typing annotations get stripped out as far as I understand, so those do not end up occupying any additional space once compiled to mpy.

Reply:
I'm really not sure how code is run on devices, but ``assert`` statements are ignored when using the -o flag or setting ``__debug__`` environment variable to ``False``.